source/tasks/CreateOctopusRelease/CreateOctopusReleaseV4/task.json

```diff
@@ -169,6 +169,24 @@
             "helpMarkDown": "Output from the deployment will appear in the log. If checked, the task will only succeed if the deployment is successful.",
             "groupName": "deployment"
         },
+        {
+            "name": "GitRef",
+            "type": "string",
+            "label": "Git Reference",
+            "defaultValue": "",
+            "required": false,
+            "helpMarkDown": "Git branch reference to use when creating the release for version controlled Projects.",
+            "groupName": "versionControl"
```

Comment:
You need to add this `group` into the `groups` array further up (L20) of this file otherwise they will not show up in the UI

e.g.
```
{
    "name": "versionControl",
    "displayName": "Version Control",
    "isExpanded": false
}
```

Reply:
nice catch. and thats why we do testing :)